elm-format-lib/src/Parse/Number.hs

```diff
@@ -262,12 +263,12 @@ chompHexInt pos end =
 -- Return -2 if it has BAD digits
 
 {-# INLINE chompHex #-}
-chompHex :: Ptr Word8 -> Ptr Word8 -> (# Ptr Word8, Int #)
+chompHex :: (Integral i) => Ptr Word8 -> Ptr Word8 -> (# Ptr Word8, i #)
```

Comment:
`chompHex` is used in `Parse.String` when parsing Unicode escapes. Changing from `Int` to `Int64` here would therefore spill over into `Parse.String`, `Elm.String` and `Reporting.Error.Syntax`. This is why this function has been given this more general type signature instead.

Not sure this is the right thing to do though. Other options could be just letting the type change spill over or use `fromIntegral` in to convert back to `Int` in `Parse.String`